src/components/cart/CheckRecipe.tsx

```diff
@@ -0,0 +1,136 @@
+import { Box, Flex, Image, Text } from "@chakra-ui/react";
+import { AddIcon, MinusIcon, SmallCloseIcon } from "@chakra-ui/icons";
+import { FC } from "react";
+import { useMyStore } from "../store/store.tsx";
+import { NewRecipe, Product } from "../types.ts";
+import { useGetRecipePrice } from "../hooks/useGetRecipePrice.tsx";
+
+type Props = {
+  recipeInCart: string;
+};
+
+type IngredientData = null | {
+  productsByStoreId: {
+    [storeId: string]: Product[];
+  };
+  ingredientIds: {
+    [storeId: string]: string[];
+  };
+};
+
+const CheckRecipe: FC<Props> = ({ recipeInCart }) => {
+  const { recipes } = useMyStore();
+  const findRecipeById = (recipeInCart: string) =>
+    recipes.find((oneRecipe) => oneRecipe.id === recipeInCart);
+
+  // @ts-ignore
+  const specificRecipe: NewRecipe = findRecipeById(recipeInCart);
```

Comment:
Jakej duvod je nenechat to dedit to naprimo z te fuknce, to je idealni? Zaroven ted trochu kecame, protoze neresime stav, kdy by `recipes.find` nic nenaslo (muze to vratit i undefined). Muzem to treba resit tak, ze v takovej pripad bychom z te funkce `findRecipeById` vyhodili chybu. Tzv. bys me se tvarili, ze se to nema stavat. Klidne proberme na callu. :)

```suggestion
  const specificRecipe = findRecipeById(recipeInCart);
```